backend/src/logging.ts

```diff
@@ -1,8 +1,9 @@
 import winston from 'winston';
 
+const logLevels = { error: 0, warn: 1, info: 2, debug: 3 };
+
 const logger = winston.createLogger({
-  level: process.env.WINSTON_LOG_LEVEL,
-  transports: [new winston.transports.Console()],
+  levels: logLevels,
+  transports: [new winston.transports.Console({ level: process.env.MAX_LOG_LEVEL })],
```

Comment:
I noticed that you added MAX_LOG_LEVEL in the destructuring assignment of process.env in config.ts so I think it's better if you add MAX_LOG_LEVEL to the config object in config.ts then import config in this file.

Reply:
Good idea. Done!